crates/aptos-rosetta/src/account.rs

```diff
@@ -195,23 +192,38 @@ async fn get_balances(
     }
 }
 
+/// Only coins supported by Rosetta
+///
+/// TODO: Allow passing in known supported coins in some sort of config file
+pub static SUPPORTED_COINS: Lazy<Vec<TypeTag>> = Lazy::new(|| vec![native_coin_tag()]);
+
 /// A cache for currencies, so we don't have to keep looking up the status of it
 #[derive(Debug)]
 pub struct CoinCache {
```

Comment:
What if we make a CoinCache instance with Lazy::new. We then don't have to pass it around. Don't know if it is idiomatic in Rust.